templates/resources/codegurureviewer_repository_association.md.tmpl

```diff
@@ -0,0 +1,33 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### S3 as the repository to associate CodeGuru Reviewer with.  The bucket name must start with the prefix codeguru-reviewer-*
+
+{{ tffile (printf "examples/resources/%s/codegurureviewer_repository_association_s3.tf" .Name)}}
+
+tags["codeguru-reviewer"]
```

Comment:
```suggestion
```

I dont believe this particular line is needed, @quixoticmonk can you confirm?

Reply:
You are right. Thanks for catching that. The note in the markdown should handle that tag reference. Was copied over by mistake.